src/components/strategies/create/CreateStrategyOption.tsx

```diff
@@ -99,17 +99,21 @@ export const CreateStrategyOption: FC<Props> = ({ base, quote }) => {
                 aria-selected={id === selectedId}
                 onClick={() => setItem(id)}
                 className={cn(
-                  'rounded-10 text-14 flex size-full flex-col items-center justify-start gap-8 bg-black px-8 py-16 outline-white/60',
+                  'rounded-10 text-14 group flex size-full flex-col items-center justify-start gap-8 bg-black px-8 py-16 outline-white/60',
                   'md:px-12',
                   'focus-visible:outline focus-visible:outline-1',
-                  id === selectedId ? 'outline outline-1 outline-white' : ''
+                  id === selectedId
+                    ? 'outline outline-1 outline-white'
+                    : 'hover:outline-background-400 hover:outline hover:outline-1'
                 )}
                 data-testid={id}
               >
                 {svg}
                 <span
                   className={`text-12 md:text-14 ${
-                    id === selectedId ? 'text-white' : 'text-white/40'
+                    id === selectedId
```

Comment:
can you use `group-aria-selected-hover:text-white/80` ?

Reply:
Added